pkg/locodedb/calls.go

```diff
@@ -62,14 +62,26 @@ func Get(locodeStr string) (Record, error) {
 
 	return Record{
 		Country:    country,
-		Location:   locodeSubstr(mLocodes[n].locationName),
-		SubDivName: locodeSubstr(mLocodes[n].subDivName),
-		SubDivCode: locodeSubstr(mLocodes[n].subDivCode),
+		Location:   locFromCSV(&mLocodes[n]),
+		SubDivName: divNameFromCSV(&mLocodes[n]),
+		SubDivCode: divCodeFromCSV(&mLocodes[n]),
 		Point:      mLocodes[n].point,
 		Cont:       mLocodes[n].continent,
 	}, nil
 }
 
-func locodeSubstr(ol offLen) string {
-	return locodeStrings[int(ol.offset) : int(ol.offset)+int(ol.length)]
+func nameFromCSV(c *locodesCSV) string {
```

Comment:
It avoids copying for sure.

Reply:
that's not an optimization since most of compilers inline such functions. And pointers may affect GC. So if we don't have clear benchmarks showing optimization with pointers, let's avoid them

Reply:
For the record, pointer-less version:
```
name    old time/op    new time/op    delta
Get-16     431ns ± 1%     455ns ± 1%  +5.47%  (p=0.000 n=10+9)

name    old alloc/op   new alloc/op   delta
Get-16     0.00B          0.00B         ~     (all equal)

name    old allocs/op  new allocs/op  delta
Get-16      0.00           0.00         ~     (all equal)
```